tempesta_fw/cfg.c

```diff
@@ -888,6 +888,73 @@ tfw_cfg_map_enum(const TfwCfgEnum mappings[],
 EXPORT_SYMBOL(tfw_cfg_map_enum);
 
 /**
+ * Get value of attribute with name @attr_key.
+ * Return @default_val if the attribute is not found in the entry @e.
+ */
+const char *
+tfw_cfg_get_attr(const TfwCfgEntry *e, const char *attr_key,
+		 const char *default_val)
```

Comment:
I see this a lot in the code. From readability point of view, it's easier on an eye when this is written in way where the second line is longer.
```c
tfw_cfg_get_attr(const TfwCfgEntry *e,
                 const char *attr_key, const char *default_val)
```

When some arguments at the end of the list are connected or similar/uniform, then an alternative is something like this:
```c
tfw_cfg_get_attr(const TfwCfgEntry *e, const char *attr_key,
                                       const char *default_val)
```

Both are cleaner and better readable, as well as easier and faster understood.